shell/src/main/java/alluxio/cli/fs/command/CopyFromLocalCommand.java

```diff
@@ -37,7 +43,19 @@
    */
   public CopyFromLocalCommand(FileSystemContext fsContext) {
     super(fsContext);
-    mCpCommand = new CpCommand(fsContext);
+    // The copyFromLocal command needs its own filesystem context because we overwrite the
```

Comment:
Do we want a specific configuration value for this, or we could make a default here and let callers of the CLI override it with an option? One issue with this is they may also want to specify other configuration options which are associated with the copy from local block write policy.